.changeset/giant-students-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blitzjs/generator": patch
+---
+
+Update form paths when running blitz generate all
```

Comment:
let's be more specific in changelogs and PR titles. This should be **fix** because it's broken :)